src/main/java/com/github/stefvanschie/inventoryframework/Gui.java

```diff
@@ -185,13 +183,38 @@ public void setRows(int rows) {
         }
 
         //copy the viewers
-        List<HumanEntity> viewers = new ArrayList<>(inventory.getViewers());
+        List<HumanEntity> viewers = getViewers();
 
         this.inventory = Bukkit.createInventory(this, rows * 9, getTitle());
 
         viewers.forEach(humanEntity -> humanEntity.openInventory(inventory));
     }
 
+    /**
+     * Gets the count of {@link HumanEntity} instances that are currently viewing this GUI.
+     *
+     * @return the count of viewers
+     * @since 0.5.19
+     */
+    @Contract(pure = true)
+    public int getViewerCount() {
+        return inventory.getViewers().size();
+    }
+
+    /**
+     * Gets a mutable snapshot of the current {@link HumanEntity} viewers of this GUI.
+     * This is a snapshot (copy) and a not a view, therefore modifications aren't visible.
```

Comment:
"and a not a view" this sounds like a typo, I think the first "a" should be removed from that sentence.